alvr/dashboard/src/steamvr_launcher/linux_steamvr.rs

```diff
@@ -77,55 +78,118 @@ pub fn linux_hardware_checks() {
                 || adapter.get_info().device_type == wgpu::DeviceType::IntegratedGpu
         })
         .map(|adapter| match adapter.get_info().vendor {
-            0x10de => DeviceInfo::Nvidia,
-            0x1002 => DeviceInfo::Amd {
-                device_type: adapter.get_info().device_type,
-            },
-            0x8086 => DeviceInfo::Intel {
-                device_type: adapter.get_info().device_type,
-            },
-            _ => DeviceInfo::Unknown,
+            0x10de => (adapter, DeviceInfo::Nvidia),
+            0x1002 => (
+                adapter,
+                DeviceInfo::Amd {
+                    device_type: adapter.get_info().device_type,
+                },
+            ),
+            0x8086 => (
+                adapter,
+                DeviceInfo::Intel {
+                    device_type: adapter.get_info().device_type,
+                },
+            ),
+            _ => (adapter, DeviceInfo::Unknown),
```

Comment:
you can simplify this a little bit, assign the match result to a variable `vendor` and then return the tuple `(adapter, vendor)`

Reply:
done

Reply:
There is an extra new line in the middle if the match branches, and maybe add a new line just before the tuple

Reply:
fixed compilation, this too